lib/miq_smis_client.rb

```diff
@@ -682,14 +682,7 @@ def self.dotNodeInfo(node)
     end
 
     def self.dotHealth(node)
-      return 'springgreen3' # XXX
```

Comment:
It's storage code that we haven't used in some time. Ultimately, we need to determine what to do with it. Until then, we should try to keep it intact. If removing that first return helps the metrics, I'm OK with it.